atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -334,7 +334,11 @@ def _get_email_from_koji_obj(self, obj):
         else:
             if not self.email_domain:
                 raise RuntimeError("Empty email_domain specified")
-            return '@'.join([obj['name'], self.email_domain])
+            elif obj.get('name'):
+                return '@'.join([obj['name'], self.email_domain])
+            else:
+                self.log.info('Koji object name is missing')
+                return None
```

Comment:
If I understand the code path correctly, if the obj has no key `name`, raising an error should be the right way to ignore the obj. Otherwise, if return None, it will cause None is included in receivers list and will be passed to `sendmail`.

BTW, is there a test case covering the case of this change?

Reply:
Good catch, thanks!
I checked test_sendmail and looks like there is no coverage for this